From 7b06ec15564def94bedf4330638da436aae1b822 Mon Sep 17 00:00:00 2001
From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Tue, 18 Mar 2025 07:42:33 +0000
Subject: [PATCH 03/11] tests: Raise the timeout on monitor-unit from 1 minute
 to 5 minutes

The test is huge and on a very fast machine it takes 10 seconds. So in
Debian and Ubuntu builds we're finding a lot of machines/architectures
where it regularly requires more than a minute to complete.

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/4340>
---
 src/tests/meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/tests/meson.build b/src/tests/meson.build
index 570c853fa..c6478fedf 100644
--- a/src/tests/meson.build
+++ b/src/tests/meson.build
@@ -265,6 +265,7 @@ test_cases += [
   {
     'name': 'monitor-unit',
     'suite': 'backend',
+    'timeout': 300,
     'sources': [ 'monitor-unit-tests.c', ],
     'depends': [
       test_client,
@@ -879,7 +880,7 @@ foreach test_case: test_cases
     env: test_env,
     depends: test_depends,
     is_parallel: false,
-    timeout: 60,
+    timeout: test_case.get('timeout', 60),
   )
 endforeach
 
-- 
2.47.1

